### ISSUE TITLE
`make dep-ensure` error

### ISSUE THREAD

**Issue by ulziibuyan** [felipemfp/sinonimos-cli#18]
The Makefile line: ```@cd ${PROJECT_PATH} $(DEP) ensure -v``` yields
```
cd /home/uerdenebileg/github.com/sinonimos-cli/src/sinonimos cd /home/uerdenebileg/github.com \
/sinonimos-cli/src/sinonimos && GOPATH=/home/uerdenebileg/github.com/sinonimos-cli dep ensure -v
```
and running make gives this error:
```
/bin/sh: line 0: cd: too many arguments
make: *** [Makefile:16: dep-ensure] Error 1
```
because ```DEP=cd $(PROJECT_PATH) && GOPATH=$(GOPATH) dep``` line already has ```cd``` command. Should one of it be removed?

I'm using GNU Make on Linux.

**Comment by felipemfp** [comment]
You're right!

Could you send a pull request fixing this error? 

I think we could remove `cd` command from `dep-ensure` and `dep-update` rules

**Comment by felipemfp** [comment]
Fixed on #19
